src/style.css

```diff
@@ -1,3 +1,74 @@
+
 body {
   margin: 0;
+  background-color: #e5e5e5;
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  max-width: 1200px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.logo__img {
+  margin-top: 10px;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__list {
+  display: flex;
+  gap: 20px;
+  margin: 0 50px;
+  padding: 0;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  position: relative;
+  padding: 0;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
```

Comment:
add `line-height` value from figma. And it'd be better to add `padding: 23px 0;`